x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -58,32 +25,53 @@ this {es} cluster.
 Keystore that contains the key and certificate for the HTTP layer for this node.
 
 `transport_keystore_all_nodes.p12`::
-Keystore that contains the key and certificate for the transport layer for all the nodes in your cluster.
---
+Keystore that contains the key and certificate for the transport layer for all
+the nodes in your cluster.
+
+You can then start {kib} and enter the enrollment token, which is valid for 30
+minutes. This token automatically applies the security settings from your {es} 
+cluster, authenticates to {es} with the `kibana_system` user, and writes the 
+security configuration to `kibana.yml`. 
+
+A security certificate is generated in the {kib} `/data` directory. This
+file establishes trust between {kib} and the {es} Certificate Authority (CA) for
+the HTTP layer.
+
+[discrete]
+=== Prerequisites
+
+* https://www.elastic.co/downloads/elasticsearch#preview-release[Download] and
+unpack the `elasticsearch 8.0.0-beta` package distribution for your
+environment.
+* https://www.elastic.co/downloads/kibana#preview-release[Download] and unpack
+the `kibana 8.0.0-beta` package distribution for your environment.
 
 [discrete]
 [[stack-start-with-security]]
 === Start {es} and enroll {kib} with security enabled
-. From the installation directory, start {es}. Passwords for the `elastic` and
-`kibana_system` users are generated and output to the terminal.
+
+. From the installation directory, start {es}. A password is generated for the 
+`elastic` user and output to the terminal, plus an enrollment token for
+enrolling {kib}.
 +
 [source,shell]
 ----
 bin/elasticsearch
 ----
 +
-TIP: You might need to scroll back a bit in the terminal to view the passwords.
+TIP: You might need to scroll back a bit in the terminal to view the password
+and enrollment token.
 
-. Copy the generated passwords and save them in a secure location. The passwords
-are shown only when you start {es} for the first time.
+. Copy the generated password and enrollment token and save them in a secure
+location. These values are shown only when you start {es} for the first time.
 +
-NOTE: If you need to reset the password for the `elastic` or the `kibana_system` user, run the
-<<reset-password,`elasticsearch-reset-password`>> tool,
-available in the {es} `/bin` directory.
+NOTE: If you need to reset the password for the `elastic` or `kibana_system`
```

Comment:
don't mention `kibana_system` user as it is not used in the enrollment process by default ( we use a service account ) . Maybe "for the `elastic` or any other built-in user" ?

Reply:
I'll change this to read, "If you need to reset the password for the `elastic` user or other built-in users"